client/app/components/composites/SearchBar/SearchBar.css

```diff
@@ -1,5 +1,5 @@
-@custom-media --SearchBar_small (max-width: 40rem);
-@custom-media --SearchBar_medium (min-width: 40rem);
+@custom-media --SearchBar_small (max-width: 62rem);
+@custom-media --SearchBar_medium (min-width: 62rem);
```

Comment:
I think there should be generic Topbar break points that all the components within it are using. These media queries were mean't to be temporary (sorry for not documenting that in the code).

As for using rems, the breakpoints will behave differently when the Topbar is used within the main app, custom landing pages, and the Styleguide. However, ems are problematic as well since the font size changes in the breakpoints. Should we just use pixels in the breakpoints?